packages/node/example/bootstrappedApp/README.md

```diff
@@ -0,0 +1,33 @@
+# Demo: authenticated a NodeJS script
+
+This demo shows how you can run an authenticated script to interact with your Pod.
+Check out the code in `src/app.js` for more details.
+
+Note that until [this issue](https://github.com/solid/node-solid-server/issues/1533)
+is resolved, this does not work against NSS, and only works against a compliant
+OIDC implementation, such as Inrupt's ESS (e.g. deployed at `https://pod.inrupt.com`).
+
+## Installing the app
+
+To use the app, first run `npm ci` in the root repository.
```

Comment:
Seeing how it imports from `../../../dist`, I presume you'd also need to run `npm run build` or something in the root?

Reply:
Good point, the instructions weren't clear on this. Running `npm ci` at the root of the repo will trigger the build, but then you'd still have to `npm ci` the demo app itself. I clarified this in 3738e8c9f8c559d8d371ff59f0d6642cbd27e6fa.

Reply:
Ah btw, about that: do we want this to fail for people following instructions if we forget to update the lockfile? (i.e. is it really better for people to run `npm ci` rather than `npm install`?) Is it likely that it will cause inexplicable errors when running this if not?